content/outcomes/application-observability/tsm-app-observability.md

```diff
@@ -0,0 +1,15 @@
+---
+date: '2021-07-13'
+lastmod: '2021-07-13'
+layout: single
+team:
+  - Mark Schweighardt
+  - Niran Even-Chen
+  - Andrew Huffman
+title: Tanzu Service Mesh Application Observability Introduction
```

Comment:
Is this file really necessary? It seems the rest of the files have a direct line to a customer outcome, but this is more just introductory text. Can we remove this and add whatever intro is necessary in the other pages?